roles/hydra-stack/install/tasks/main.yml

```diff
@@ -6,7 +6,7 @@
 #
 - include: tomcat.yml
 - include: solr_5.yml
-  when: solr_version | match("^5\.[0-9]\.[0-9]")
+  when: solr_version | match("^5\.\d{1,2}\.{1,2}")
```

Comment:
I also noticed the second digit check is missing \d in the commit. Adding that fix as well with the next commit.